src/gen/conditions.rs

```diff
@@ -474,6 +474,15 @@ pub const ELIGIBLE_FOR_DEDUP: u32 = 1;
 // If the spend bundle contained *any* relative seconds or height condition, this flag is set
 pub const HAS_RELATIVE_CONDITION: u32 = 2;
 
+// If the CoinSpend is eligible for fast-forward, this flag is set. A spend is
```

Comment:
The original idea was that since it involves reviewing each output coin, it's cheaper to do it when we build the list of outputs instead of a separate pass. However, I ended up deferring this check to perform it only if all other conditions are met.
The only remaining reason to keep these checks here is because of the `condition_counter`. The order the conditions are created in is lost in this function, and the only valid position for `ASSERT_MY_PARENT_ID` is as the second condition. This can't really be checked outside of this function. The order could be recorded perhaps, but it's not obvious that's an overall simplification.